insight.py

```diff
@@ -85,7 +85,7 @@ def collector(self):
         # call `collector` and store data to output dir
         base_dir = os.path.join(util.pwd(), "../")
         collector_exec = os.path.join(base_dir, "bin/collector")
-        collector_outdir = fileutils.create_dir(
+        collector_outdir = fileopt.create_dir(
```

Comment:
define collector_outdir in front of `__init__` of in `__init__`

Reply:
We don't necessarily run `collector` every time, it's not needed if we don't call `collector()`